notte/browser/context.py

```diff
@@ -16,8 +16,15 @@ def interaction_nodes(self) -> list[InteractionNode]:
     def markdown_description(self) -> str:
         return self.format(self.node, indent_level=0)
 
-    def format(self, node: NotteNode, indent_level: int = 0) -> str:
+    def format(
+        self,
+        node: NotteNode,
+        indent_level: int = 0,  # indentation level for the current node.
```

Comment:
I think these should probably be computed when the Notte graph is created intead of when it is formated in markdown. In my other PR I created a new function`NotteNode.from_a11y_node` and you can use that to build the necessary variables.

Reply:
Hmm yes and no. For `_path` I agree, all other variables are strictly correlated with the `format` function as the number of chars depends on the way you build your resulting output string. There is no way for me to predict this ahead of time (before calling `format()` I mean)

Reply:
yeah that's right actually